### DISCUSSION TITLE
Changing tags to something else?

### DISCUSSION THREAD

**Discussion by Porges** [kylechui/nvim-surround#283] [Q&A]
With `vim-surround` it’s possible (for example) to turn `<b>word</b>` into `*word*` via `cst*`. However with nvim-surround I just get prompted for a tag name, so it turns into `<*>word</*>`. Is there a way to achieve this, or is this a bug?

**Comment by kylechui** [comment]
You can achieve this via custom setup; it is not possible using the default behavior.
